File: codeforces/197/a.c
#include <stdio.h>

int main(int argc, char* argv[]) {
	int a, b, c, win = 1;
	scanf("%d %d %d", &a, &b, &c);
	c += c;
	printf("%s", a >= c && b >= c ? "First" : "Second");
	return 0;
}

File: codeforces/197/a.kt
import java.util.*

fun main(args: Array<String>) {
	val s = Scanner(System.`in`)
	val a = s.nextInt()
	val b = s.nextInt()
	val c = s.nextInt() * 2
	if (a >= c && b >= c) {
		println("First")
	} else {
		println("Second")
	}
}

